### DISCUSSION TITLE
How to Filter To Lowercase?

### DISCUSSION THREAD

**Discussion by bogordesaincom** [spatie/laravel-query-builder#808] [Q&A]
I'm use Postgresql when i create custom filter like this :

```php
<?php
....
    public function __invoke(Builder $query, $value, string $property): Builder
    {
        return $query->where('id', 'LIKE', '%'.$value.'%')
            ->orWhere(function (Builder $query) use ($value) {
                return $query->where('name', 'LIKE', '%'.$value.'%');
            });
    }
```
When Searching `Name`  results is fine, but if i searching `name` this can't get results

**Comment by abkrim** [comment]
Use 'ILKE' instead or read this about [citext — a case-insensitive character string type](https://www.postgresql.org/docs/current/citext.html#CITEXT) Tip
